KlaviyoSwift.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "KlaviyoSwift"
-  s.version          = "3.2.0"
```

Comment:
cocoapods are not yet setup correctly for release since it's needing a actual tag on the repo. For now did something to get CI happy but once I publish a tag and SPM I'll fix cocoapods.